src/SFA.DAS.EAS.Web/Views/EmployerCommitments/TransferDetails.cshtml

```diff
@@ -0,0 +1,120 @@
+        <hr class="hr-top" />
+        <div class="column-one-third">
+            Cohort Reference
+        </div>
+        <div class="column-half">
+            @Model.Data.HashedCohortReference
+        </div>
+    </div>
+    <div class="column-full">
+        <hr class="hr-top" />
+        <div class="column-one-third">
+            Status
+        </div>
+        <div class="column-two-thirds">
+            Pending
+        </div>
+    </div>
+    <div class="column-full">
+        <hr class="hr-top" />
+        <div class="column-one-third">
+            Total Cost
+        </div>
+        <div class="column-two-thirds">
+            @($"£{Model.Data.TotalCost:n0}")
+        </div>
+    </div>
+    <div class="column-full">
+        <hr class="hr-top" />
+        <div class="column-one-third">
+            Training
+        </div>
+        <div class="column-two-thirds">
+            @foreach (var item in Model.Data.TrainingList)
+            {
+               <p>@item.SummaryDescription</p>
+            }
+        </div>
+        <hr class="hr-bottom" />
+    </div>
+</div>
+
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <form method="POST" action="@Url.Action("ApproveTransfer")" onsubmit="sfa.tagHelper.submitRadioForm('@ViewBag.Title')">
+            
+            @Html.Partial("ValidationSummary", Html.ViewData.ModelState)
+
+            <fieldset role="radiogroup" aria-required="true">
+                <div class="form-error-group form-group @Html.AddClassIfPropertyInError("Confirmation", "error")">
+                    <h1 class="heading-medium" for="">Do you want to approve this transfer request?</h1>
+                    <div>
+                        @Html.ValidationMessage("Confirmation", "Please choose an option", new { id = "error-message-Confirmation" })
+                    </div>
+                    <label class="block-label selection-button-radio" for="Confirmation-True">
+                        @Html.RadioButton("Confirmation", "True", new { id = "Confirmation-True", dataOptionName = "Yes", onclick = $"sfa.tagHelper.radioButtonClick('{ViewBag.Title}', 'Yes')" })
+                        Yes, use approve the request
+                    </label>
+                    <label class="block-label selection-button-radio" for="Confirmation-False">
+                        @Html.RadioButton("Confirmation", "False", new { id = "Confirmation-False", dataOptionName = "No", onclick = $"sfa.tagHelper.radioButtonClick('{ViewBag.Title}', 'No')" })
+                        No, reject the request
+                    </label>
+                </div>
+            </fieldset>
+            
+            <button type="submit" class="button" id="submit-transfer-connection">Continue</button>
+            <a class="links" href="@Url.Action("Index", "EmployerCommitments")" aria-label="Cancel">Cancel</a>
+
+        </form>
+    </div>
+</div>
+@section breadcrumb {
+    <div class="breadcrumbs">
```

Comment:
Where should the back and cancel links go? I would say they should jump to the transfers dashboard on MYA:  {myabase/accounts/{HashedAccountId}/transfers) via the ExternalUrl helper.

Reply:
At the moment, tsince we don't have a bingo box. I've sent them back to the index page.